src/plots/polar/polar.js

```diff
@@ -1187,6 +1196,9 @@ proto.updateAngularDrag = function(fullLayout) {
         var fullLayoutNow = _this.gd._fullLayout;
         var polarLayoutNow = fullLayoutNow[_this.id];
 
+        dx *= fullLayout._inverseScaleX;
+        dy *= fullLayout._inverseScaleY;
+
         var x1 = x0 + dx;
         var y1 = y0 + dy;
```

Comment:
Would it work to just do this conversion in calculating `x1` and `y1`? It always makes things more confusing when a variable changes meaning, and I don't see `dx` and `dy` getting used after this.
```suggestion
        var x1 = x0 + dx * fullLayout._inverseScaleX;
        var y1 = y0 + dy * fullLayout._inverseScaleY;
```